README.md

```diff
@@ -31,11 +31,11 @@ If you're interested in how the project is organized at a higher level, please c
 ## Our PoC team ❤️
 
 Developers
-| [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy)
+| [<img src="https://github.com/YlanGarnier.png?size=85" width=85><br><sub>[Ylan GARNIER]</sub>](https://github.com/YlanGarnier) | [<img src="https://github.com/paulogarithm.png?size=85" width=85><br><sub>[Paul PARISOT]</sub>](https://github.com/paulogarithm) | [<img src="https://github.com/Saverio976.png?size=85" width=85><br><sub>[Xavier MITAULT]</sub>](https://github.com/Saverio976)
 | :---: | :---: | :---: |
 
 Manager
-| [<img src="https://github.com/adrienfort.png?size=85" width=85><br><sub>[Manager's name]</sub>](https://github.com/adrienfort)
```

Comment:
You can remove the brackets around the names:\
`Ylan GARNIER` instead of `[Ylan GARNIER]` :wink:

Reply:
done :slightly_smiling_face: